app/routes/questionnaire.py

```diff
@@ -338,6 +338,13 @@ def get_confirmation_email_sent():
 
     email = URLParamSerializer().loads(request.args.get("email"))
 
+    try:
+        ConfirmationEmail()
+    except ConfirmationEmailLimitReached:
+        show_send_another_email_guidance = False
+    else:
+        show_send_another_email_guidance = True
+
```

Comment:
```suggestion
	show_send_another_email_guidance = True

    try:
        ConfirmationEmail()
    except ConfirmationEmailLimitReached:
        show_send_another_email_guidance = False
```

Reply:
This has changed in the latest changes, it now calls a static method to determine this instead of initialising a ConfirmationEmail which initialised unnecessary attributes when limit hasn't been reached. I have also refactored feedback to be consistent with email.